src/shared/src/backend-supports/projects-get-all.ts

```diff
@@ -12,7 +12,12 @@ const calculateEndDate = (start: Date, weeks: number) => {
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+interface WP {
+  startDate: any;
```

Comment:
The goal for this ticket is to remove **any** type in projects-get-all.ts. Can you replace the data type of startDate with the correct type? (Srry if this confused you last night, I just wrote **any** for this field on your laptop to quickly show you an property example we need for the work package)